docker/test/develop.yml

```diff
@@ -0,0 +1,42 @@
+# This needs at least compose 1.6.0
+version: '2'
+
+services:
+  postgresdata:
+    image: postgres:9.5
+    command: /bin/true
+
+  postgres:
+    image: postgres:9.5
+    volumes_from:
+      - postgresdata
+    environment:
+      POSTGRES_PASSWORD: myhuginnpassword
+      POSTGRES_USER: huginn
+
+  huginn_test_postgres:
```

Comment:
here I was trying to follow the somewhat established pattern of using [`develop.yml` to denote a docker-compose file used for building/running a local a docker image](https://github.com/huginn/huginn/blob/master/docker/single-process/develop.yml#L24) vs. using [`docker-compose.yml` for pulling/running a remote docker image](https://github.com/huginn/huginn/blob/master/docker/single-process/docker-compose.yml#L20). I think it would make sense to build on this branch to include Docker support for running Huginn in `RAILS_ENV=development` mode, but I'd personally like to limit the scope of this change to `RAILS_ENV=test` if that's alright.